src/relay/transforms/dynamic_to_static.cc

```diff
@@ -33,7 +34,9 @@ namespace relay {
 class DynamicToStaticMutator : public MixedModeMutator {
  public:
   DynamicToStaticMutator()
-      : dyn_reshape_op_(Op::Get("dyn.reshape")), dyn_tile_op_(Op::Get("dyn.tile")) {}
+      : dyn_reshape_op_(Op::Get("dyn.reshape")),
```

Comment:
In this case, I think it is probably okay to  not cache them. Otherwise, we may have a long list here and a long list of members. This would have some lookup overhead, but it increases the readability. How do you think?

Reply:
I'm thinking a Registry will be the best combination, we can do a dynamic lookup to see if the op is registered, simplify this codebase, and still have the caching optimization.

Reply:
The ops are actually cached in a global registry. This caching here is to reduce the dynamic lookup overhead.

Reply:
Next PR?

Reply:
no problem. Maybe do a cleanup PR with the MakOp stuff. I notice the duplicate as well.